src/pip/_internal/wheel.py

```diff
@@ -825,4 +857,4 @@ def build(self, requirements, session, autobuilding=False):
                 ' '.join([req.name for req in build_failure]),
             )
         # Return True if all builds were successful
```

Comment:
This comment line is no longer valid and should be removed.

Reply:
Thanks, noted.